web/beacon-app/src/features/members/components/MemberDetails.tsx

```diff
@@ -0,0 +1,69 @@
+import { Toast } from '@rotational/beacon-core';
+import { useState } from 'react';
+
+import { BlueBars } from '@/components/icons/blueBars';
+
+import { useFetchMembers } from '../hooks/useFetchMembers';
+import CancelAccount from './CancelAccount';
+
+export default function MemberDetails() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleToggleBars = () => {
+    const open = isOpen;
+    setIsOpen(!open);
+  };
+
+  const handleClose = () => setIsOpen(false);
+
+  const { member, hasMemberFailed, isFetchingMember, error } = useFetchMembers();
+
+  const { id, name, created } = member;
+
+  if (isFetchingMember) {
+    return <div>Loading...</div>;
+  }
+
+  if (error) {
+    return (
+      <Toast
+        isOpen={hasMemberFailed}
+        onClose={handleClose}
+        variant="danger"
+        title="We are unable to fetch your member, please try again."
+        description={(error as any)?.response?.data?.error}
+      />
+    );
+  }
+
+  return (
+    <>
+      <h3 className="mt-10 text-2xl font-bold">User Profile</h3>
+      <h4 className="mt-10 max-w-4xl border-t border-primary-900 pt-4 text-xl font-bold">
+        User Details
+      </h4>
+      <section className="mt-8 max-w-4xl rounded-md border-2 border-secondary-500 pl-6">
+        <div className="mr-4 mt-3 flex justify-end">
+          <BlueBars onClick={handleToggleBars} />
+          <div>{isOpen && <CancelAccount close={handleClose} />} </div>
+        </div>
+        <div className="flex gap-16 pt-4 pb-8">
+          <h6 className="font-bold">User Name:</h6>
+          <span className="ml-3">{name}</span>
+        </div>
+        {/*         <div className="flex gap-12 pb-8">
```

Comment:
Email address isn't in the model for members. It can be discussed if it should be added in sprint 4.